src/windows/models/files_model.py

```diff
@@ -343,7 +344,8 @@ def add_files(self, files, image_seq_details=None, quiet=False, prevent_image_se
                 # Let the event loop run to update the status bar
                 get_app().processEvents()
                 # Update the recent import path
-                settings.setDefaultPath(settings.actionType.IMPORT, dir_path)
+                if not prevent_recent_folder:
```

Comment:
Kinda reminds me of taking the opposite of an opposite.  Like a double negative,  Nothing wrong with it just seems like it would be cleaner to say something like "use_recent_folder". Struggling to find the exact right words here.